### PR TITLE
feat: build wasm using SRTool

### PR DESCRIPTION
**Pull request by weichweich**
## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -64,3 +64,23 @@ build:
...
+build-wasm:
+  image:
+    name: paritytech/srtool:1.53.0
+    entrypoint: [""]
+  stage: build
+  only:
+    - develop
+    - master
+  script:
+    - export PACKAGE=peregrine-runtime
```
Comment (wischli) [inline .gitlab-ci.yml:77]: I suppose we also want the `spiritnet-runtime`?

File: .maintain/reset-spec/app.py [hunk 2]
```diff
@@ -108,44 +122,46 @@ def make_native(docker_img, out_file, chain, runtime):
     tmp_dir = os.path.join("/tmp/", str(uuid.uuid1()))
     os.mkdir(tmp_dir)
 
-    if args.westend:
```
Comment (Diiaablo95) [inline .maintain/reset-spec/app.py:111]: We still have the `--westend` option but we don't check anymore for it?

Comment (weichweich) [reply .maintain/reset-spec/app.py:111]: good catch! Fixed in 31fa889

File: .gitlab-ci.yml (hunk 1 above)
Comment (weichweich) [reply .gitlab-ci.yml:77]: fixed in d104b68

**Pull request merged**